server/planning/search/queries/combined.py

```diff
@@ -22,8 +22,12 @@ def construct_combined_view_data_query(
 
     query = elastic.ElasticQuery()
 
-    if len(search_filter["params"]):
-        search_dates(search_filter["params"], query)
+    filter_params = common.get_params_from_search_filter(search_filter)
+    if len(filter_params):
+        filter_params["time_zone"] = params.get("time_zone")
+        filter_params["start_of_week"] = params.get("start_of_week")
```

Comment:
I wonder how this will be populated, especially when using those search filters for scheduled exports. imo those values are not stored in search filters atm

Reply:
What about add a `time_zone` input for scheduled exports form, and use app config for `start_of_week` on search?

Reply:
we have also default timezone config, I would probably start with that, not sure if we have a need to use different timezones in those exports